doc/user/load_save.md

```diff
@@ -523,6 +523,77 @@ outImages.push_back("bandicoot-favicon-inv.jpeg");
 mlpack::data::Save(outImages, matrix, info);
 ```
 
+### Resize images
+
+It is possible to resize images in mlpack with the following function:
+
+- `ResizeImages(images, info, newWidth, newHeight)`
+   * `images` an armadillo matrix contains a set of images, each one is
+     represeted in one column representing the image as a flattened vector.
+
+   * `info` will be populated with information from the image in `image`.
+
+   * `newWidth` the new width of the resized image.
+
+   * `newHeight` the new height of the resized image.
+
+   * This function does not return but it modifies the info object and images
+     matrix.
```

Comment:
```suggestion
   * This function returns `void` and modifies `info` and `images`.
```
Seems more concise, up to you.  (At the very least backticks around info/images would be good.)